package.json

```diff
@@ -189,6 +189,8 @@
 		"codemirror": "^5.56.0",
 		"coffeescript": "^2.5.1",
 		"connect": "^3.7.0",
+		"cookie": "^0.4.1",
```

Comment:
I don't think this dependency is part of the problem you're solving.

Reply:
@renatobecker I have made the changes requested. 2 workflows require approval.